test/Microsoft.AspNet.Security.Test/project.json

```diff
@@ -18,7 +20,8 @@
     "frameworks": {
         "aspnet50": {
             "dependencies": {
-                "Shouldly": "1.1.1.1"
+                "Shouldly": "1.1.1.1",
+                "System.Security.Claims": "1.0.0-*"
```

Comment:
Oh.. I thought of saying System.Security.Claims be moved to parent dependencies node. That should work on both clrs as desktop version will type forward.